azure/terraform/app.tf

```diff
@@ -0,0 +1,19 @@
+resource "azuread_application_registration" "ternary" {
+  display_name     = "Ternary2"
```

Comment:
```suggestion
  display_name     = "Ternary"
```

I assume this was a temp change to avoid conflict with the existing registered application while developing this TF module?

Reply:
ah yep, good eye - fixed